packages/openbridge-webcomponents/src/automation/automation-badge/automation-badge.css

```
@@ -44,7 +44,7 @@
 
   &.regular {
     @mixin style style=normal noClick;
```

Stylelint (17.9.0)

[error] Expected mixin name "style style=normal noClick" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)